Makefile

```diff
@@ -45,7 +52,9 @@ install: ## Build and install the binary with the current source code. Use it to
 
 .PHONY: generate
 generate: ## Update/generate all mock data. You should run this commands to update the mock data after your changes.
-	make generate-setup
+	# if an controller-gen be locally installed already it will be removed to ensure that the generate will
+	# get and use the correct version defined in the testdata makefiles instead of an previous installation.
```

Comment:
This statement doesn't match what is done.
controller-gen is not installed after removing.

Reply:
HI @mengqiy.

See: 

<img width="503" alt="Screenshot 2020-04-13 at 17 55 26" src="https://user-images.githubusercontent.com/7708031/79140653-37aca980-7db0-11ea-9bed-ce4dbfd232a4.png">

And then: 

<img width="611" alt="Screenshot 2020-04-13 at 17 55 53" src="https://user-images.githubusercontent.com/7708031/79140662-3a0f0380-7db0-11ea-92e4-147a8420c9c9.png">

Reply:
Ha... You rely on the Makefile in the generated project to install the controller-gen for you.
I didn't realized it. It's not intuitive, but seems to be OK.

Reply:
Please update the comment here to reflect how it work.
Then it should be OK to go.

Reply:
Let me know if you are ok with the updates in the comments.